proofread secure your platform (#2556)
centreon/centreon-documentation: versioned_docs/version-23.10/administration/secure-platform.md

```shell
Status for the jail: centreon
|- Filter

> |  |- Currently failed:	1.

Reviewer: ```suggestion
|  |- Currently failed:	1
```

> |  |- Total failed:	17.

Reviewer: ```suggestion
|  |- Total failed:	17
```

|  `- File list:	/var/log/centreon/login.log
`- Actions

>    |- Currently banned:	0.

Reviewer: ```suggestion
   |- Currently banned:	0
```

>    |- Total banned:	2.

Reviewer: ```suggestion
   |- Total banned:	2
```

  openssl genrsa -aes256 2048 > ca_demo.key
  ```
  
5. Create an x509 certificate from the private key of the certificate of certification authority:
  
  Create an x509 certificate that will be valid for one year: **ca_demo.crt** in our case.
  
  > Note that it is necessary to simulate a trusted third party, so the **Common Name** field must be different from the server certificate.
  ```text
  openssl req -new -x509 -days 365 -key ca_demo.key -out ca_demo.crt
  ```
  

>   The certificate being created will enable you to sign your server certificate.

Reviewer: ?

>   The certificate being created will enable you to sign your server certificate.

Reviewer: ```suggestion
  Once you have created the certificate, you will be able to use it to sign your server certificate.
```

systemctl status apache2
```

If everything is ok, you should have:

```shell
● apache2.service - The Apache HTTP Server
    Loaded: loaded (/lib/systemd/system/apache2.service; enabled; vendor pres>

>      Active: active (running) since Tue 8/9/2022 5:01:36 AM UTC; 3h 56min ago

Reviewer: ```suggestion
     Active: active (running) since Tue 2022-08-09 05:01:36 UTC; 3h 56min ago
```

<TabItem value="Alma / RHEL / Oracle Linux 8" label="Alma / RHEL / Oracle Linux 8">

```shell

> vim /etc/httpd/conf.d/10-centreon.conf

Reviewer: ```suggestion
vi /etc/httpd/conf.d/10-centreon.conf
```

```

</TabItem>
<TabItem value="Alma / RHEL / Oracle Linux 9" label="Alma / RHEL / Oracle Linux 9">

```shell

> vim /etc/httpd/conf.d/10-centreon.conf

Reviewer: ```suggestion
vi /etc/httpd/conf.d/10-centreon.conf
```

```

</TabItem>
<TabItem value="Debian 11" label="Debian 11">

```shell

> vim /etc/apache2/sites-available/centreon.conf

Reviewer: ```suggestion
vi /etc/apache2/sites-available/centreon.conf
```

## Enabling http2

> It is possible to enable the http2 protocol to improve Centreon network performance.

Reviewer: ```suggestion
It is possible to enable the http2 protocol to improve Centreon's network performance.
```
